testdata/map_spin_lock.c

```diff
@@ -0,0 +1,23 @@
+/* This file excercises bpf_spin_lock. */
```

Comment:
You need to add this file to TARGETS in Makefile and rebuild the testdata via `make` using the docker build image.

Reply:
I just did but no idea why CI shows: generated files are not up to date

Reply:
Fixed!